tools/api_key_tool.py

```diff
@@ -6,35 +6,46 @@
 import argparse
 
 fixtures_directory = 'tests/fixtures/'
+
+# restclient api header configuration
 zero_api_regex = r'(\[Token )0{40}(\])'
 real_api_regex = r'(\[Token ).{40}(\])'
 zero_token_string = '[Token ' + 40 * '0' + ']'
 
+# pandas json api call configuration
+pd_real_api_regex = r'&token=.{40}'
```

Comment:
If we end up scrubbing the API key from the URL, we won't need to maintain 2 separate scrubbers